From: Alexander Larsson <alexl@redhat.com>
To: linux-fsdevel@vger.kernel.org
Cc: linux-kernel@vger.kernel.org, gscrivan@redhat.com,
	david@fromorbit.com, brauner@kernel.org, viro@zeniv.linux.org.uk,
	Alexander Larsson <alexl@redhat.com>
Subject: [PATCH v3 1/6] fsverity: Export fsverity_get_digest
Date: Fri, 20 Jan 2023 16:23:29 +0100	[thread overview]
Message-ID: <f5f292caee6b288d39112486ee1b2daef590c3ec.1674227308.git.alexl@redhat.com> (raw)
In-Reply-To: <cover.1674227308.git.alexl@redhat.com>

Composefs needs to call this when built in module form, so
we need to export the symbol. This uses EXPORT_SYMBOL_GPL
like the other fsverity functions do.

Signed-off-by: Alexander Larsson <alexl@redhat.com>
---
 fs/verity/measure.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/fs/verity/measure.c b/fs/verity/measure.c
index 5c79ea1b2468..875d143e0c7e 100644
--- a/fs/verity/measure.c
+++ b/fs/verity/measure.c
@@ -85,3 +85,4 @@ int fsverity_get_digest(struct inode *inode,
 	*alg = hash_alg->algo_id;
 	return 0;
 }
+EXPORT_SYMBOL_GPL(fsverity_get_digest);
-- 
2.39.0
